query/src/main/scala/filodb/query/exec/aggregator/RowAggregator.scala

```diff
@@ -121,8 +121,8 @@ object RowAggregator {
   def apply(aggrOp: AggregationOperator, params: Seq[Any], schema: ResultSchema): RowAggregator = {
     val valColType = ResultSchema.valueColumnType(schema)
     aggrOp match {
-      case Min      => MinRowAggregator
-      case Max      => MaxRowAggregator
+      case Min if valColType == ColumnType.DoubleColumn => MinRowAggregator
```

Comment:
Do you see any case where `LongColumn` and `IntColumn` would need this? I understand we don't want to use this for histograms but by restricting to DoubleColumn only we also don't support Long and Int columns